### PR TITLE
Add first openshift billing metrics tests

### PR DESCRIPTION
**Pull request by jgarciao**
This PR add a couple of tests for OpenShift Monitoring Metrics (ODS-173 and ODS-175). I've added them the Sanity tag

Please note that the code still has to be refactored not to include a copy of "Iterative Image Test" keyword once all JupyterHub keywords are refactored too

### CODE REVIEW

**Comment by jgarciao** [comment]
https://opendatascience-jenkins-csb-rhods.apps.ocp4.prod.psi.redhat.com/job/rhods-ci-pr-test/61/

I think there are the usual test fails about python version and others. 

The tests related to metrics passed:
```
==============================================================================
Tests.Monitor And Manage                                                      
==============================================================================
Tests.Monitor And Manage.Metrics                                              
==============================================================================
Tests.Monitor And Manage.Metrics.Billing Metrics                              
==============================================================================
Verify OpenShift Monitoring results are correct when running undef... | PASS |
------------------------------------------------------------------------------
Test Billing Metric (notebook cpu usage) on OpenShift Monitoring      Current CPU usage: 0.24985176646000004
| PASS |
------------------------------------------------------------------------------
Tests.Monitor And Manage.Metrics.Billing Metrics                      | PASS |
2 tests, 2 passed, 0 failed
```

**Review by vasukulkarni — APPROVED** [review]

**Comment by vasukulkarni** [comment]
We have merge commit in the history, can you rebase without merge commits? Thanks

**Comment by jgarciao** [comment]
I think I got rid of the merge commit properly

**Comment by jgarciao** [comment]
https://opendatascience-jenkins-csb-rhods.apps.ocp4.prod.psi.redhat.com/job/rhods-ci-pr-test/66/console

Again, I think the usual tests failed:
```
==============================================================================
Tests.Monitor And Manage.Metrics                                              
==============================================================================
Tests.Monitor And Manage.Metrics.Billing Metrics                              
==============================================================================
Verify OpenShift Monitoring results are correct when running undef... | PASS |
------------------------------------------------------------------------------
Test Billing Metric (notebook cpu usage) on OpenShift Monitoring      Current CPU usage: 0.44316571561291673
| PASS |
------------------------------------------------------------------------------
Tests.Monitor And Manage.Metrics.Billing Metrics                      | PASS |
2 tests, 2 passed, 0 failed
==============================================================================
Tests.Monitor And Manage.Metrics                                      | PASS |
2 tests, 2 passed, 0 failed
==============================================================================
Tests.Monitor And Manage                                              | PASS |
2 tests, 2 passed, 0 failed
==============================================================================
Tests.Jupyterhub                                                              
==============================================================================
Tests.Jupyterhub.Image-Iteration                                              
==============================================================================
Open ODH Dashboard                                                    | PASS |
------------------------------------------------------------------------------

Iterative Testing Classifiers                                         | FAIL |
Several failures occurred:

1) 'Python 3.8.3' does not match 'Python 3.8.6'

2) Element 'id:progress-bar' did not disappear in 10 minutes.
------------------------------------------------------------------------------

Iterative Testing Clustering                                          | FAIL |
Several failures occurred:

1) Element 'xpath://div[@class="pf-c-card__title" and .="JupyterHub"]' not visible after 10 seconds.

2) Element 'xpath://div[@class="pf-c-card__title" and .="JupyterHub"]' not visible after 10 seconds.

3) Element 'xpath://div[@class="pf-c-card__title" and .="JupyterHub"]' not visible after 10 seconds.

4) Element 'xpath://div[@class="pf-c-card__title" and .="JupyterHub"]' not visible after 10 seconds.
------------------------------------------------------------------------------
Tests.Jupyterhub.Image-Iteration                                      | FAIL |
```

File: tests/Resources/Page/OCPDashboard/Monitoring/Metrics.robot [hunk 1]
```diff
@@ -0,0 +1,40 @@
+*** Settings ***
+Library  SeleniumLibrary
+
+*** Variables ***
+${METRICS_QUERY_TEXTAREA}  xpath=/html/body/div[2]/div[1]/div/div/div/div/main/div/div/div/section/div/div[2]/div[2]/div[2]/div/div[3]/div/div[1]/textarea
+${METRICS_QUERY_RESULTS_PARENT_ELEMENT}            xpath=/html/body/div[2]/div[1]/div/div/div/div/main/div/div/div/section/div/div[2]/div[2]/div[2]/div/div[3]/div[2]
+${METRICS_QUERY_RESULTS_TABLE}                     xpath=/html/body/div[2]/div[1]/div/div/div/div/main/div/div/div/section/div/div[2]/div[2]/div[2]/div/div[3]/div[2]/table
+${METRICS_QUERY_RESULTS_TABLE_ROW1_VALUE_ELEMENT}  xpath=/html/body/div[2]/div[1]/div/div/div/div/main/div/div/div/section/div/div[2]/div[2]/div[2]/div/div[3]/div[2]/table/tbody/tr/td[4]
```
Comment (LaVLaS) [inline tests/Resources/Page/OCPDashboard/Monitoring/Metrics.robot:8]: This is likely to break with the smallest change to the page structure. Is there any way to simplify this where you search for a parent div by title, class or id?

Comment (jgarciao) [reply tests/Resources/Page/OCPDashboard/Monitoring/Metrics.robot]: I've simplified the locators in today's commit

**Comment by jgarciao** [comment]
Arthy tested this PR today and all billing metric tests passed. There where the usual JupyterHub known errors

https://opendatascience-jenkins-csb-rhods.apps.ocp4.prod.psi.redhat.com/job/rhods-ci-pr-test/71


```
==============================================================================
Tests                                                                         
==============================================================================
Tests.Monitor And Manage                                                      
==============================================================================
Tests.Monitor And Manage.Metrics                                              
==============================================================================
Tests.Monitor And Manage.Metrics.Billing Metrics                              
==============================================================================
Verify OpenShift Monitoring results are correct when running undef... | PASS |
------------------------------------------------------------------------------
Test Billing Metric (notebook cpu usage) on OpenShift Monitoring      | PASS |
------------------------------------------------------------------------------
Tests.Monitor And Manage.Metrics.Billing Metrics                      | PASS |
2 tests, 2 passed, 0 failed
==============================================================================
Tests.Monitor And Manage.Metrics                                      | PASS |
2 tests, 2 passed, 0 failed
==============================================================================
Tests.Monitor And Manage                                              | PASS |
2 tests, 2 passed, 0 failed
```

**Comment by jgarciao** [comment]
This is being tested right now again in https://opendatascience-jenkins-csb-rhods.apps.ocp4.prod.psi.redhat.com/job/rhods-ci-pr-test/86/

RHODS: 1.0.16

**Comment by jgarciao** [comment]
The CUDA build failed because of "500 Internal Server Error" pushing the image to the docker registry. I'm running it again here:  https://opendatascience-jenkins-csb-rhods.apps.ocp4.prod.psi.redhat.com/job/rhods-ci-pr-test/87

**Pull request merged**